src/iio_wrapper.cpp

```diff
@@ -102,43 +102,28 @@ IIOWrapper::IIOWrapper()
     m_local_context = iio_create_local_context();
     if (!m_local_context) throw std::runtime_error("Exception: local context is null");
 
-    int count = iio_context_get_devices_count(m_local_context);
-    if (count > 0) {
-      struct iio_device * dev = iio_context_get_device(m_local_context, 0);
-      const char * name = iio_device_get_name(dev);
-      RCLCPP_INFO(rclcpp::get_logger("rclcpp_iiowrapper"), "device name: %s", name);
-
-      IIOWrapper::s_device_name = std::string(name);
-
-      std::list<std::string> adis1650x_x{"adis16505-1", "adis16505-2", "adis16505-3", "adis16500",
-                                         "adis16507-1", "adis16507-2", "adis16507-3"};
-
-      bool found =
-        (std::find(adis1650x_x.begin(), adis1650x_x.end(), IIOWrapper::s_device_name) !=
-         adis1650x_x.end());
-      if (found) {
-        IIOWrapper::s_device_name_enum = IIODeviceName::ADIS1650X;
+    std::list<std::string> adis165{"adis16505-1", "adis16505-2", "adis16505-3", "adis16500",
```

Comment:
please add devices alphabetically (adis16500 should be first)
please rename adis165 to supported_devices